pkg/controller/defaults/volumeclass.go

```diff
@@ -54,3 +76,24 @@ func addVolumeClassDefaults(ctx context.Context, c kclient.Client, app *v1.AppIn
 
 	return nil
 }
+
+func getDefaultVolumeSize(ctx context.Context, c kclient.Client, appInstance *v1.AppInstance) (v1.Quantity, error) {
+	// If the Status.Defaults.VolumeSize has been set, use that.
+	if appInstance.Status.Defaults.VolumeSize != nil {
+		return v1.Quantity(appInstance.Status.Defaults.VolumeSize.String()), nil
+	}
+
```

Comment:
nit: this function is only used when we know that the `VolumeSize == nil`. Therefore, this check is redundant. Maybe not that big of a deal because, based on other comments here, `VolumeSize` is going to be removed.

Reply:
Yep, that's the idea. Right now it is not longer being set in this PR. I still want it in the schema to allow old apps to move over and then I'll remove it at that point.

Reply:
Update - I did go ahead and remove this check though as I agree with your assessment.